src/client/core-client.ts

```diff
@@ -63,11 +67,14 @@ interface StopServiceReply {
 }
 
 interface DeployServiceRequest {
```

Comment:
A CLI/UI can be made using these API, of course not really the plan right now but let's keep this open for now, it's not much work and the goal is to generate this anyway so that will be easier to generate all and not having to do specific filters